server/app/models/ApiKey.java

```diff
@@ -94,6 +94,11 @@ public Instant getExpiration() {
     return expiration;
   }
 
+  /** True if the supplied time is after the expiration date. */
+  public boolean expiredAfter(Instant instant) {
```

Comment:
? Do we need to message anything about the key being UTC and this should be too?

Reply:
updated the method doc